README.md

```diff
@@ -10,29 +10,40 @@ Try `example/app.js` with
 npm install
 npm start
 
-open localhost:8080/user1/bundle.js
-open localhost:8080/user2/bundle.js
+open localhost:8080/user1
+open localhost:8080/user2
 ```
 
 ## Usage
 
 ```js
 const multiUserDevServer = require('multi-user-dev-server');
-const app = multiUserDevServer(user => `/home/${user}/repo/webpack.config.js`);
+
+const app = multiUserDevServer(username => {
+  return {
+    // The path to this user's webpack config
+    configPath: `${__dirname}/${username}/webpack.config.js`,
+    // The `env` to pass into the webpack config
+    webpackEnv: {},
+    // What to respond with for `GET /:username` (optional)
+    successResponse: `Bundle completed in ${__dirname}/${username}`,
+  };
+});
+
 app.listen(8080);
 ```
 
 ## Server API
 
-Get a bundle
+Start building `username`'s webpack bundle. If it's already building, this will wait to respond until building is complete.
 
 ```
-GET /:username/:bundle-name
+GET /:username
 ```
 
 
-Reload one user's webpack config
+Reload `username`'s webpack config.
 
 ```
-POST /reload/:username
```

Comment:
Hmm. Maybe leave this as `POST /reload/...`?

If theres a good reason not to below, disregard. POSTing a username to index looks weird.

Reply:
I thought `POST /:username` seemed more consistent with the `GET /:username` endpoint, since they perform similar actions.

(`GET` lazy-loads the config if it hasn't been loaded yet, whereas `POST` forces a reload of the config.)

I don't have a strong preference. What do you think?

Reply:
Gotcha. That does make more sense.

I would suggesting leaning towards POST only. But for this purpose GET seems fine.
I think to be really explicit, you could change this to be `POST /load/:username` and `POST /reload/:username` with explanations? But that's really just an opinion. Nothing to block on.

Reply:
I think you're right. `POST` makes more sense for both.